Src/Common/W3C/W3CConstants.cs

```diff
@@ -0,0 +1,93 @@
+﻿namespace Microsoft.ApplicationInsights.W3C
+{
+    using System;
+    using System.ComponentModel;
+
+    /// <summary>
+    /// W3C constants.
+    /// </summary>
+    [Obsolete("Not ready for public consumption.")]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+#if DEPENDENCY_COLLECTOR
+    public
+#else
+    internal
+#endif
+    static class W3CConstants
+    {
+        /// <summary>
+        /// W3C traceparent header name.
+        /// </summary>
+        public const string TraceParentHeader = "traceparent";
+
+        /// <summary>
+        /// W3C tracestate header name.
+        /// </summary>
+        public const string TraceStateHeader = "tracestate";
+
+        /// <summary>
+        /// Name of the field that carry ApplicationInsights application Id in the tracestate header.
+        /// </summary>
+        public const string ApplicationIdTraceStateField = "@msappid";
```

Comment:
recommendation is to add `@` for the vendor name in case of multi-tenant. For a single tenant - this delimiter is not needed.

Reply:
Also for azure we might need more than just app-id

Reply:
ok, so do we need to add @ now? do we know what else we need right now?